flask/app/models.py

```diff
@@ -69,6 +69,7 @@ class Participant(db.Model):
     sex: Sex = db.Column(db.Enum(Sex), nullable=False)
     # Sample.SampleType
     participant_type: ParticipantType = db.Column(db.Enum(ParticipantType), nullable=False)
+    month_of_birth: str = db.Column(db.DateTime)
```

Comment:
Just use [`db.Date`](https://docs.sqlalchemy.org/en/13/core/type_basics.html#sqlalchemy.types.Date) since the time is irrelevant. The Python typing should be `datetime.date`

Reply:
resolved in d86688a